staking-cli/src/info.rs

```diff
@@ -12,30 +12,49 @@ pub async fn stake_table_info(
     l1_block_number: u64,
 ) -> Result<Vec<Validator<BLSPubKey>>> {
     let l1 = L1Client::new(vec![l1_url])?;
-    let st = l1
-        .get_stake_table(stake_table_address, l1_block_number)
+    let validators = l1
+        .fetch_all_validators(stake_table_address, l1_block_number)
         .await?;
-    Ok(st
+
+    Ok(validators
         .into_iter()
         .map(|(_address, validator)| validator)
         .collect())
 }
 
-pub fn display_stake_table(stake_table: Vec<Validator<BLSPubKey>>) -> Result<()> {
+pub fn display_stake_table(stake_table: Vec<Validator<BLSPubKey>>, compact: bool) -> Result<()> {
     let mut stake_table = stake_table.clone();
     stake_table.sort_by(|a, b| a.stake.cmp(&b.stake));
 
     for validator in stake_table.iter() {
         let comm: Commission = validator.commission.try_into()?;
         let bls_key = validator.stake_table_key.to_string();
-        let end = bls_key.chars().map(|c| c.len_utf8()).take(40).sum();
+        let key_str = if compact {
+            let end = bls_key.chars().map(|c| c.len_utf8()).take(40).sum();
+            format!("{}..", bls_key[..end].to_string())
+        } else {
+            bls_key.to_string()
+        };
         tracing::info!(
-            "Validator {}: {}... comm={} stake={} ESP",
+            "Validator {}: {key_str} comm={comm} stake={} ESP",
             validator.account,
-            &bls_key[..end],
-            comm,
             format_ether(validator.stake),
         );
+
+        if validator.delegators.is_empty() {
+            tracing::info!(" - No delegators");
+            continue;
+        }
+
+        // sort delegators by address for easier reading
```

Comment:
shouldn't sort be by stake?

Reply:
Thought about it but I just think it's easier to find stuff like this.